Assets/Scripts/Dialogue/ActivateTextAtLine.cs

```diff
@@ -34,7 +34,7 @@ public class ActivateTextAtLine : MonoBehaviour {
 
     void OnTriggerEnter2D(Collider2D other)
     {
-		if (other.CompareTag ("Player") || other.CompareTag ("Doll")) 
+		if (other.CompareTag ("Player")) 
```

Comment:
Wouldn't this break the conveyor belt level because dolls who the player are not possessing can land on buttons?